LanguageFeatures/Records/members_A04_t04.dart

```diff
@@ -85,9 +85,9 @@ main() {
   1010, 1011, 1012, 1013, 1014, 1015, 1016, 1017, 1018, 1019, 1020, 1021,
       1022, 1023, 1024, 1025);
 
-  Expect.equals(0, r.$0);
-  Expect.equals(101, r.$101);
-  Expect.equals(999, r.$999);
-  Expect.equals(1024, r.$1024);
-  Expect.equals(1025, r.$1025);
+  Expect.equals(0, r.$1);
+  Expect.equals(101, r.$102);
+  Expect.equals(999, r.$1000);
+  Expect.equals(1024, r.$1025);
+  Expect.equals(1025, r.$1026);
```

Comment:
It seems reasonable to test the treatment of large records. Perhaps there could be a bug in the handling of 4 digit `$` names, or in the treatment of `$N` where `N` is a decimal numeral whose binary representation has more than 10 bits, etc.

With that in mind we should probably cover a couple of values _around_ the potential critical values, like `$999`, `$1000`, `$1001`, `$1023`, `$1024`, `$1025`, `$1026`, and `Expect.throws` on `$1027`.